allennlp/modules/seq2vec_encoders/bert_pooler.py

```diff
@@ -54,7 +54,14 @@ def get_input_dim(self) -> int:
     def get_output_dim(self) -> int:
         return self._embedding_dim
 
-    def forward(self, tokens: torch.Tensor, mask: torch.BoolTensor = None):
-        pooled = self.pooler(tokens)
+    def forward(
+        self, tokens: torch.Tensor, mask: torch.BoolTensor = None, num_wrapping_dims: int = 0
+    ):
+        pooler = self.pooler
+        for _ in range(num_wrapping_dims):
+            from allennlp.modules import TimeDistributed
+
+            pooler = TimeDistributed(pooler)
```

Comment:
Jesus Christ that formatting

Reply:
You mean the blank line? Yeah, that is unfortunate, but you could just have the import at the top.

Reply:
Local imports are faster because they are lazy, but more importantly they are never circular. The only problem is that they blow up your functions like this in the formatting.